ui/plugins/controller.js

```diff
@@ -278,37 +278,50 @@ treeherder.controller('PluginCtrl', [
             }
         };
 
-        $scope.retriggerJob = function() {
+        $scope.retriggerJob = function(jobs) {
             if ($scope.user.loggedin) {
```

Comment:
I could send this function either list of jobs or list of job_ids. I think sending it a list of jobs and then extracting whatever information we want is better. Let me know if I should just send it a list of job_ids.